app/controllers/StoriesVisibleController.scala

```diff
@@ -22,7 +22,11 @@ class StoriesVisibleController(
   def storiesVisible(containerType: String) =
     AccessAPIAuthAction(parse.json[StoriesVisibleRequest]) { implicit request =>
       val storiesVisible =
-        containerService.getStoriesVisible(containerType, request.body.stories)
+        containerService.getStoriesVisible(
+          containerType,
+          request.body.stories,
+          collectionConfigJson = null
```

Comment:
If this is a valid situations where `collectionConfigJson` is not provided can you make the field optional and pass `None` instead of `null`?